designs/virtualization/4.x/Templates/Templates.md

```diff
@@ -0,0 +1,57 @@
+# Templates
+
+## Virtual machines and VM templates are located under Virtualization
+
+Removing the tab ‘Virtual Machine Templates’ from the side navigation and combining it with ‘Virtual Machines’, so both are located under ‘Virtualization’.
+This PR will focus on how users can create a VM when they choose a to create a VM from a Template.
+
+### Virtual Machine Templates Tab
+
+From the Virtualization left nav item the user goes to the Virtual Machine Templates Tab where they can see the full list of templates.
+From this point, we suggest 2 options to create a VM:
+
+- Create VM from the selected templated via an action from the kebab menu
+
+![TemplatesTabOp1](img/TemplatesTabsOp1.png)
+
+- Create VM from the selected templated via an action from the kebab menu and also allow an action link
+
+![TemplatesTabOp2](img/TemplatesTabsOp2.png)
+
+Both options will open a modal where the user will need to provide a name for the VM they want to create.
+Users can click the button link ‘View Template details’ to view the Template details page.
+Clicking the secondary button ‘Edit and Create’ will allow them to edit the template before they create the VM.
+
+![ModalToProvideName](img/NoMissingInfo.png)
+
+In case a user selects a template that has required info missing, they will be taken to the wizard to edit the details and then create the VM. An inline alert will be shown and indicate which info is missing.
+
+![Back to wizard to complete info](img/Step-1-basic-template-3.png)
```

Comment:
The placement of the inline alert feels a bit odd here. I wonder if we should just put it in the content area of the wizard. Also for the list maybe have it General : Name, Storage : Flavor to indicate where to find the field.

Reply:
I think that if it will be in the wizard area, it will have to be shown in every section, while when it's above - it stays constant.